standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/cache/CacheUtils.java

```diff
@@ -144,4 +147,15 @@ public static boolean matches(String name, String pattern) {
     }
     return false;
   }
+
+  public static boolean validateShouldCacheTableCanUseEventIsActiveTransaction(String catName, String dbName,
+      String tblName, boolean canUseEvents, RawStore rawStore) {
+    return !shouldCacheTable(catName, dbName, tblName) || (canUseEvents && rawStore.isActiveTransaction());
+  }
+
+  public static boolean validateShouldCacheTableCanUseEventIsActiveTransactionIsConstraintsValid(String catName,
```

Comment:
The method name is too long and not readable. Shall make it shouldGetConstraintFromRawStore().

Reply:
done